src/platforms/stm32/src/lib/gpio_driver.c

```diff
@@ -128,49 +113,49 @@ enum gpio_pin_state
 };
 
 static const AtomStringIntPair gpio_bank_table[] = {
-    { ATOM_STR("\x1", "a"), GPIOA },
-    { ATOM_STR("\x1", "b"), GPIOB },
-    { ATOM_STR("\x1", "c"), GPIOC },
-    { ATOM_STR("\x1", "d"), GPIOD },
-    { ATOM_STR("\x1", "e"), GPIOE },
-    { ATOM_STR("\x1", "f"), GPIOF },
-    { ATOM_STR("\x1", "g"), GPIOG },
-    { ATOM_STR("\x1", "h"), GPIOH },
+    { a_atom, GPIOA },
```

Comment:
Let's take `a_atom` as an example, that is used just here at this line.
I don't see any practical vantage in defining it elsewhere, on the other side I need to jump to other places when doing refactoring, renaming them to something else, adding new ones (then we have to be consistent) etc...
Also, I don't feel like we are improving much readability since `ATOM_STR("\x1", "a")` tells very well what kind of thing we are creating and what is its value.

But maybe I'm not considering other aspects, so please share your point here so we can take a decision about the style that we should keep.

Reply:
These have been reverted to keep the original format.